### DISCUSSION TITLE
Row height configuration [Jun 11]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2425] [Discord Threads]
> 🙋‍♂️ **λRick**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@198536651348115466> Sorry, currently it's not configurable. We recently changed row height to ensure image preview is substantial enough. We understand it would be better to keep it configurable. Will discuss with the team  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/984649831316348928)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
